openedx/features/personalized_learner_schedules/call_to_action.py

```diff
@@ -103,6 +116,22 @@ def _make_reset_deadlines_cta(xblock, is_learning_mfe=False):
                              'any of your progress.'),
         }
 
+        if category == cls.CAPA_SUBMIT_DISABLED and xblock.attempts:
```

Comment:
We only want to show this when the user has already attempted the problem. If they haven't attempted, we will show the default shift your dates so we don't use "Try again" language when they haven't tried yet.

The case of they used all their attempts is covered above where we don't show the CTA regardless.